src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -173,7 +175,7 @@ public SingleRun(DocumentDatabase database, RavenConfiguration configuration, Sc
                 ScriptEngine.SetValue("Raven_Min", new ClrFunctionInstance(ScriptEngine, Raven_Min));
                 ScriptEngine.SetValue("Raven_Max", new ClrFunctionInstance(ScriptEngine, Raven_Max));
 
-                ScriptEngine.SetValue("convertJsTimeToTimeSpanString", new ClrFunctionInstance(ScriptEngine, ConvertJsTimeToTimeSpanString));
```

Comment:
This means that old client libraries will stop working due to the lack of this method?

Reply:
I think so, I believe we need to keep it around.